app/views/teaching_events/index.html.erb

```diff
@@ -6,23 +6,22 @@
       All events listed here are designed to help you get into teacher training. They are all free to attend.
     </p>
 
-    <details>
-      <summary>What are the event types?</summary>
-
-      <dl>
+    <dl>
+      <div>
         <dt>DfE Train to Teach events</dt>
+        <dd>Watch presentations, hear from teachers and speak to advisers one-to-one.</dd>
+      </div>
 
-        <dl>Watch presentations, hear from teachers and speak to advisers one-to-one.</dl>
-
+      <div>
         <dt>DfE Online Q&amp;As</dt>
+        <dd>Get answers to your specific questions from an online panel.</dd>
+      </div>
 
-        <dl>Get answers to your specific questions from an online panel.</dl>
-
+      <div>
         <dt>Training provider events</dt>
-
-        <dl>Find out about local courses and how to apply; hosted by teacher training providers.</dl>
-      </dl>
-    </details>
+        <dd>Find out about local courses and how to apply; hosted by teacher training providers.</dd>
+      </div>
+    </dl>
```

Comment:
@Tomcowan1, @EllieNodder - any thoughts on this text? They just provide an overview of the event types and what's involved:

![Screenshot from 2021-11-29 11-21-40](https://user-images.githubusercontent.com/128088/143859248-ab6fdf75-bbb6-4bd8-a4de-b0f6e0592951.png)